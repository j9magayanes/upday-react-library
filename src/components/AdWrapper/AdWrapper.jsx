import React from 'react';
import './adwrapper.css';

export const AdWrapper = ({ children }) => {
    return (
      <div className="ad-wrapper grow">
        {children}
      </div>
    )
}