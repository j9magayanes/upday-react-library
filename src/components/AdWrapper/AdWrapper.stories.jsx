import React from 'react';

import { AdWrapper } from './AdWrapper';


export default {
  title: 'Form/AdWrapper',
  component: AdWrapper,
};

const Template = () =><AdWrapper/>;


export const Default = Template.bind({});