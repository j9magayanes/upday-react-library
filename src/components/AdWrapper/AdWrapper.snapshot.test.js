import React from 'react';
import renderer from 'react-test-renderer';
import { AdWrapper } from './AdWrapper';

test('renders AdWrapper component correctly', () => {
  const component = renderer.create(<AdWrapper>Test Content</AdWrapper>);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
