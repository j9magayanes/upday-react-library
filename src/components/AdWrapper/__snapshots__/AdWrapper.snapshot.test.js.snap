// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AdWrapper component correctly 1`] = `
<div
  className="ad-block grow"
>
  Test Content
</div>
`;
